frontend/cypress/pages/LoginPage.js

```diff
@@ -1,3 +1,5 @@
+/// <reference types="cypress" />
```

Comment:
is this neccessary ?

Reply:
Absolutely, it enhances the developer experience by providing auto-suggestions while writing test cases.

Reply:
hi @Ankit-Matth, possibly we have different extensions/versions in our vscode, but for me I can already get cypress suggestions while I type. When you set this option does this get applied globally, or just for the LoginPage.js?